README.md

```diff
@@ -0,0 +1,61 @@
+# Dev environemnt setup
+
+## Terms 
```

Comment:
I'd add
- shell - the (usually) textual interface to the letting system, i.e. CMD, bash, PowerShell, ash, fish, zsh...
- terminal - the application / window in which some shell runs.